app/services/get_stormglass_api_service.rb

```diff
@@ -10,18 +10,22 @@ def call
     lat = @spot.latitude;
     lng =  @spot.longitude;
     source = 'sg,meteo,noaa';
-    end_time = Time.now.utc.beginning_of_day + 7.days;
+    end_time = Time.now.utc.beginning_of_day + 1.days;
```

Comment:
i might need you to explain this to me. I thought this line defines the number of days we call the API for